credit-advisory-schema/src/main/resources/ddl/init.sql

```diff
@@ -0,0 +1,61 @@
+-- Define enum types
+CREATE TYPE advisor_role AS ENUM ('associate', 'partner', 'senior');
+CREATE TYPE phone_type AS ENUM ('home', 'work', 'mobile');
+CREATE TYPE application_status AS ENUM ('new', 'assigned', 'on_hold', 'approved', 'canceled', 'declined');
+
+-- Create users table
+CREATE TABLE users (
+                       id BIGSERIAL CONSTRAINT users_PK PRIMARY KEY,
+                       email VARCHAR(255) NOT NULL CONSTRAINT users_email_UQ UNIQUE CHECK (email LIKE '%@%'),
+                       username VARCHAR(255) NOT NULL
+);
+
+-- Create advisors table
+CREATE TABLE advisors (
+                          id BIGSERIAL CONSTRAINT advisors_PK PRIMARY KEY REFERENCES users,
```

Comment:
If id is referencing to another table then it should not be serial

Reply:
If constraint is PK and FK at the same time then we should also have it in constraint name